midi_app_controller/gui/_tests/test_binds_editor.py

```diff
@@ -0,0 +1,300 @@
+import pytest
+from unittest.mock import patch
```

Comment:
`unittest.mock` is in the standard library so it should be placed in a separate section.